l16/dev/genesis.json

```diff
@@ -0,0 +1,862 @@
+      "storage": {
+        "0x00000000000000000000000000000000000000000000000000000000000000":"0x7880bf33442b07d1af22347c74f6edfc0a155fb03be9f0348a553c75dac70184",
```

Comment:
In our case shouldn't we start network with `0x00000000000000000000000000000000000000000000000000000000000020` (0 deposit txs?)